stdlib/src/math.rs

```diff
@@ -831,9 +831,34 @@ mod math {
         (x.fract(), x.trunc())
     }
 
-    #[pyfunction]
-    fn nextafter(x: ArgIntoFloat, y: ArgIntoFloat) -> f64 {
-        float_ops::nextafter(*x, *y)
+    #[derive(FromArgs)]
+    struct NextAfterArgs {
+        #[pyarg(positional)]
+        x: ArgIntoFloat,
+        #[pyarg(positional)]
+        y: ArgIntoFloat,
+        #[pyarg(named, optional)]
+        steps: OptionalArg<ArgIndex>,
+    }
+
+    #[pyfunction]
+    fn nextafter(arg: NextAfterArgs, vm: &VirtualMachine) -> PyResult<f64> {
+        let steps: Option<i64> = arg
+            .steps
+            .map(|v| v.try_to_primitive(vm))
+            .transpose()?
+            .into_option();
+        match steps {
+            Some(steps) => {
+                if steps < 0 {
+                    return Err(
+                        vm.new_value_error("steps must be a non-negative integer".to_string())
+                    );
+                }
+                Ok(float_ops::nextafter(*arg.x, *arg.y, Some(steps as u64)))
```

Comment:
Since nextafter call with step and without step are well separated, `float_ops::nextafter` also looks to be fine to be separated.
Because `nextafter` is originated from libc, I prefer to keep it as it is unless there is a good reason to change it.